components/var_dumper.rst

```diff
@@ -82,6 +82,15 @@ current PHP SAPI:
     #. From time to time, run ``composer global update symfony/var-dumper``
        to have the latest bug fixes.
 
+.. tip::
+
+    The VarDumper component also includes a ``dd()`` helper method. This "dump
```

Comment:
I'm not entirely happy with my reword. Let's ask our native speaker @weaverryan to review this short paragraph. Thanks!